integration/src/main/resources/configs/integration/DatasetPermissionOperations.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "DatasetPermissionOperations",
+  "description": "Dataset Permission Operations",
+  "serverSpecificationFile": "catalog-local.json",
+  "kubernetes": {},
+  "application": {},
+  "testScripts": [
+    {
+      "name": "DatasetPermissionOperations",
+      "numberOfUserJourneyThreadsToRun": 1,
+      "userJourneyThreadPoolSize": 2,
+      "expectedTimeForEach": 5,
+      "expectedTimeForEachUnit": "MINUTES"
```

Comment:
do you mean move snapshot creation to setup, and just have one snapshot shared by all the test helper functions?

Reply:
Yes, I was thinking you could narrow it down to two snapshots, one created by `admin` and one created by `user`.

You could probably use a single one since `admin` has `SET_POLICY` it can add/remove all roles before running a test. But maybe we want to let TDR manage the owner setting and not modify that in our test?

Reply:
But also yes, move the snapshot creation into `setup()`.